HCI-lab/src/components/header/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import { Link } from "gatsby"
+
+import Navigation from "../navigation"
+import BurgerMenu from "../burgerMenu"
+
+import Logo from "../images/logo"
+
+import { headerNavigationTabsLeft, headerNavigationTabsRight } from "../../constants"
+
+import styles from "./style.module.css"
+
+const Header = () => (
+    <div className={styles.headerContainer}>
+        <Navigation navigationTabs={headerNavigationTabsLeft}/>
+        <Link className={styles.logoContainer} to={headerNavigationTabsLeft[0].linkTo}>
+            <Logo/>
```

Comment:
`headerNavigationTabsLeft[0].linkTo`  bi odradia ko lokalna varijabla `let home = navigationTabs.filter(tab => tab.name == 'Home')` i onda `to={home.linkTo}`